int32 object_id
uint8 model_id
float32 score_2d
float32 score_3d
float32 score_final
geometry_msgs/Pose pose
float32 valid_flag
